nginx.conf.template

```diff
@@ -144,9 +144,25 @@ http {
         body_filter_by_lua  'ngx.ctx.buffered = (ngx.ctx.buffered or "") .. string.sub(ngx.arg[1], 1, 1000)
                             if ngx.arg[2] then ngx.var.resp_body = ngx.ctx.buffered end';
         header_filter_by_lua 'ngx.var.resp_headers = cjson.encode(ngx.resp.get_headers())';
+        
+        add_header 'Access-Control-Allow-Origin' '*';
```

Comment:
@vrde Except for the timeout change (from 20 to 12 hours), this stuff is deployed at test.ipdb.io.